app/src/main/java/ar/scacchipa/twittercloneapp/presentation/authwebdialog/AuthWebDialogViewModel.kt

```diff
@@ -1,12 +1,12 @@
-package ar.scacchipa.twittercloneapp.viewmodel
+package ar.scacchipa.twittercloneapp.presentation.authwebdialog
 
 import android.webkit.WebResourceError
 import android.webkit.WebViewClient
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import ar.scacchipa.twittercloneapp.data.ResponseDomain
+import ar.scacchipa.twittercloneapp.data.repository.ResponseDomain
 import ar.scacchipa.twittercloneapp.domain.AuthorizationUseCase
 import ar.scacchipa.twittercloneapp.domain.ConsumableAuthUseCase
 import ar.scacchipa.twittercloneapp.utils.Constants
```

Comment:
this comment apply for the whole folder "authwebdialog". This is part of the login so it should be inside the login folder and again the **naming** the name that has now, implies that is some kind of web dialog that can be reused anywhere in the app and it's not what it is at all, maybe try something like "loginWebSection" or "loginWebview" etc

Reply:
AuthWebDialog was converted to LoginWebSection